client/src/Components/Overview.jsx

```diff
@@ -87,23 +115,64 @@ class Overview extends React.Component{
             Overview 
             </div>
             <OverviewList overviews={this.state.overview}/>
-          </div> 
-        : null }
+            <br></br>
+           <hr></hr> </div>
+        : null }  
+       
+        
 
         <div className="shippingInformation">
           <div style={{fontWeight: 'bold'}}>
             Shipping & Returns
           </div>
+
+          {!this.state.enteredShippingInfo ?
+          <div>
           <div className="manufacturingTime" style={{fontWeight: 'bold'}}>
             Ready to ship in {this.state.shippingTime}-{this.state.shippingTime + 2} days
           </div>
           <div className="shippedFrom">
             From California
           </div>
-          <div className="shippingCost" style={{textDecorationLine: 'underline', cursor: "pointer"}} onClick={this.getShippingInfo}>
+          <div className="getShippingCost" style={{textDecorationLine: 'underline', cursor: "pointer"}} onClick={this.getShippingInfo}>
              Get Shipping Cost 
+          </div> </div> :
+
+          <div> 
+            <div className='deliver' style={{fontWeight: 'bold'}}>
+              Estimated delivery: Sep 10 - Sept 29
```

Comment:
will this remains hardcoded?

Reply:
I'm going to refactor my database and change it afterwards